integrationTests/node-esm/index.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable node/no-missing-import, import/no-unresolved */
```

Comment:
Please don't just duplicate `node` test here.
Please try to test ESM-specific things here (e.g. importing `package.json`, dynamic import, etc.).
Also please create a test setup where you have an intermediate package with `peerDepenedicies` for `graphql` and test how it works with ESM package.